### PR TITLE
Replace theme conditionals in templates with new skyIfTheme directive

### PR DESCRIPTION
**Pull request by johnhwhite**
- Replace theme conditionals in templates with new skyIfTheme directive

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme)

File: src/app/public/modules/flyout/flyout-iterator.component.html [hunk 1]
```diff
@@ -6,19 +6,17 @@
     class="sky-btn"
     [attr.aria-label]="'skyux_flyout_iterator_previous_button' | skyLibResources"
     [disabled]="previousButtonDisabled"
-    [ngClass]="{
-      'sky-btn-default sky-margin-inline-compact': themeName === 'default',
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/flyout/flyout-iterator.component.html:10]: We can remove `themeName` from the `.ts` file now.

File: src/app/public/modules/flyout/flyout.component.html [hunk 2]
```diff
@@ -7,9 +7,11 @@
   [id]="flyoutId"
   [ngClass]="{
     'sky-flyout-hidden': !isOpen && !isOpening,
-    'sky-flyout-fullscreen': isFullscreen,
-    'sky-shadow': themeName === 'default',
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/flyout/flyout.component.html:11]: We can remove `themeName` from the `.ts` file now.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme](https://developer.blackbaud.com/skyux-flyout?_sp.spa=pr-replace-conditionals-with-skyiftheme)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**